tests/search_current_dir/expands_~_base_dir.fish

```diff
@@ -0,0 +1,8 @@
+# mock everything so that the only thing that gets sent to stdout are the args passed into fd
+mock fd \* "echo \$argv"
+mock fzf \* ""
+mock commandline --current-token "echo ~/"
+mock commandline \* ""
+set fd_args (__fzf_search_current_dir)
+@test "~/ not in fd args" -z (string match "~/" $fd_args)
+@test "~/ is expanded to $HOME" -n (string match --entire "$HOME" $fd_args)
```

Comment:
As a coding convention, I always specify `--`:
```fish
string match --entire -- "$HOME" $fd_args
```

Reply:
Thanks Jorge! That's an important convention I will copy.